binding_generator/Config.zig

```diff
@@ -0,0 +1,23 @@
+const Config = @This();
+
+build_target: []const u8,
+extension_api: fs.File,
+gdextension_interface: fs.File,
+output: fs.Dir,
+verbosity: Verbosity,
+
+pub const Verbosity = enum {
+    quiet,
+    verbose,
+};
+
+pub fn deinit(self: *Config, allocator: Allocator) void {
+    allocator.free(self.build_target);
```

Comment:
This is a little bit of a code smell since we're not allocating this string inside Config

But it's only used inside codegen so I'll let it be.